doc/macros.rst

```diff
@@ -1,99 +0,0 @@
-Macros
-======
```

Comment:
My ignorance here, but won't removing this file remove these things from the documentation?

Reply:
They are included in another way.